test/c/run_tests.py

```diff
@@ -38,6 +38,9 @@ def test_c(name, c_opts, sail_opts, valgrind, compiler='cc'):
             basename = os.path.splitext(os.path.basename(filename))[0]
             tests[filename] = os.fork()
             if tests[filename] == 0:
+                if basename.startswith('config'):
+                    sail_opts += ' --c-include sail_config.h'
```

Comment:
Shouldn't this be included automatically?

Reply:
Not sure, the idea is to avoid breaking existing Makefiles. To include it by default everything in `lib/json` would need to be in `lib` but that would make it harder to swap out the JSON implementation.

Reply:
Ah ok fair enough. I guess this is one of those things where a --target-version, or --edition type flag would help.